### PR TITLE
Added overview of C# relational (comparison) operators

### LINKED ISSUE #10738: Abridged Reference: operators / relational operators
The relational operators are:

- `<` (less than)
- `>` (greater than)
- `<=` (less than or equal)
- `>=` (greater than or equal)

Sub-tasks are:

- [ ] Create the new articles
- [ ] Search for existing issues on content being migrated and consolidated. This is a good opportunity to fix those.
- [ ] Redirect existing content to the new articles.
- [ ] Remove old articles.
- [ ] Update all links in the .NET Content

### PR DESCRIPTION
**Pull request by pkulikov**
Fixes #10738
No updates in the dotnet/samples repo; reusing existing code samples.

### CODE REVIEW

File: docs/csharp/language-reference/operators/arithmetic-operators.md [hunk 1]
```diff
@@ -157,23 +157,6 @@ The subtraction operator `-` subtracts its second operand from its first operand
...
-
-Binary arithmetic operators are left-associative. That is, operators with the same precedence level are evaluated from left to right.
-
-Use parentheses, `()`, to change the order of evaluation imposed by operator precedence and associativity.
-
-[!code-csharp-interactive[precedence and associativity](~/samples/snippets/csharp/language-reference/operators/ArithmeticOperators.cs#PrecedenceAndAssociativity)]
-
-For the complete list of C# operators ordered by precedence level, see [C# operators](index.md).
-
 ## Compound assignment
```
[the remark this answers is not in the document]
Comment (BillWagner) [reply docs/csharp/language-reference/operators/arithmetic-operators.md]: I think that works best. Good change.

File: docs/csharp/language-reference/operators/relational-operators.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+---
+title: "Relational operators - C# Reference"
```
[the remark this answers is not in the document]
Comment (BillWagner) [reply docs/csharp/language-reference/operators/relational-operators.md]: I think "comparison operators" is the best choice, but we'll measure once it's live.  Here's what I found:

- Google search traffic indicates "comparison operator" is more popular than "relational operator". That implies that "comparison operator" would be better.
- In the docs repo, we've used both "relational operators" and "comparison operators" almost the same number of times.

File: docs/csharp/language-reference/operators/relational-operators.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+The `>=` operator returns `true` if its first operand is greater than or equal to its second operand, `false` otherwise:
+
+[!code-csharp-interactive[greater than or equal example](~/samples/snippets/csharp/language-reference/operators/GreaterAndLessOperatorsExamples.cs#GreaterOrEqual)]
+
+## Operator overloadability
+
+A user-defined type can [overload](../keywords/operator.md) the `<`, `>`, `<=`, and `>=` operators.
+
+If a type overloads one of the `<` or `>` operators, it must also overload another one. If a type overloads one of the `<=` or `>=` operators, it must also overload another one.
```
Comment (BillWagner) [inline docs/csharp/language-reference/operators/relational-operators.md]:
```suggestion
If a type overloads one of the `<` or `>` operators, it must also overload both `<` and `>`. If a type overloads one of the `<=` or `>=` operators, it must also overload both `<=` and `>=`.
```

I think that's a bit more clear.

**Review by BillWagner — CHANGES_REQUESTED** [review]
Excellent work as always @pkulikov Thank you!

I had one suggestion, and then this is ready to :shipit:

Thanks again for all the work you do. We're all very grateful for your continued participation.

File: docs/csharp/language-reference/operators/relational-operators.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (pkulikov) [reply docs/csharp/language-reference/operators/relational-operators.md]: @BillWagner thanks for research; let's start with a better version: I'll rename the file to *comparison-operators.md* and push the update.

File: docs/csharp/language-reference/operators/relational-operators.md (hunk 3 above)
Comment (pkulikov) [reply docs/csharp/language-reference/operators/relational-operators.md]: @BillWagner thanks; I didn't apply the suggestion, but addressed the feedback almost word in word.

**Comment by pkulikov** [comment]
@BillWagner I've addressed you feedback and renamed the overview to "Comparison operators". Thanks for review; it's ready for the final look.

**Review by BillWagner — APPROVED** [review]

**Comment by BillWagner** [comment]
Thanks for the quick updates @pkulikov 
I'll :shipit: now.

**Pull request merged**